src/components/policy/policy_premium/include/policy/policy_table_interface.xml

```diff
@@ -0,0 +1,230 @@
+<?xml version="1.0" standalone="no"?>
```

Comment:
@LuxoftAKutsan 
This file is not being used anymore, policy table structure defined statically now.